scheduler/classes/Driver.py

```diff
@@ -0,0 +1,10 @@
+from scheduler.classes.Member import Member
+
+class Driver(Member):
```

Comment:
I would add a `is_dues_paying` field to driver as well. I know we ignore it, but we might not always want to ignore it. It might make things more versatile in the future.

Reply:
Although in that case that field should be moved to the member class. By doing that you'd effectively eliminate the need for a rider. 

I think there is benefit in having a rider class because it makes easy to read the code. And even if the rider is just a child that is the same as member it would allow us to add attributes to a rider later on. What do you think? I think it could be worth keeping an unchanged child class of the rider

Reply:
I agree, there's a benefit of keeping it separate. Decided to keep Rider class for readability.